### PR TITLE
Add privacy bundle test - WIP

### PR DESCRIPTION
**Pull request by lesleyjanenorton**

### CODE REVIEW

File: views/upgrade.hbs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      <button class="upgrade-now upgrade-btn flx flx-col" data-page-location="upgrade-now-footer-button">
+        <span class="upgrade-now-line-1">Upgrade Now</span>
+        <span class="upgrade-now-line-2">$39.99 USD/Year</span>
+      </button>
+    </section>
+  </div>
+
+{{else}}
+  <h1>Please log in to your account to view this page</h1>
+  <button>Sign In</button>
```
Comment (groovecoder) [inline views/upgrade.hbs:116]: This should be an active sign-in button?

File: views/upgrade.hbs [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<div class="upgrade-modal close-upgrade-modal">
+  <div class="upgrade-modal-content">
+    <button class="upgrade-x-close close-upgrade-modal">Close Modal</button>
+
+  {{#unless req.session.user.waitlists_joined.privacy_bundle}}
+    <div class="upgrade-modal-join flx flx-col jst-cntr txt-cntr show-modal-panel">
+      <div class="flx flx-col jst-cntr txt-cntr">
+        <h1 class="upgrade-modal-h1">You Just Helped Us With An Experiment!</h1>
+        <h2 class="upgrade-modal-h2">Thank you for your interest in Firefox Plus Unlimited</h2>
```
Comment (groovecoder) [inline views/upgrade.hbs:9]: This needs to match "Firefox Privacy Defender" ?

File: views/upgrade.hbs (hunk 1 above)
Comment (lesleyjanenorton) [reply views/upgrade.hbs:116]: This is actually just going away since we're redirecting anyone who isn't signed in back to the homepage.

**Comment by lesleyjanenorton** [comment]
Yeah the upgrade buttons should still be visible but if you click one of them you should see the modal confirming you have been added to the Test Panel instead of the modal asking you to join it.

File: controllers/home.js [hunk 3]
```diff
@@ -116,19 +116,32 @@ function protectMyEmail(req, res) {
...
-    return {"email_relay": {"notified": false} };
+    return {"privacy_bundle": {"notified": false} };
   }
-  user.waitlists_joined["email_relay"] = {"notified": false };
+  user.waitlists_joined["privacy_bundle"] = {"notified": false };
   return user.waitlists_joined;
 }
 
-function addEmailToRelayWaitlist(req, res) {
+function addPrivacyBundleToRelayWaitlist(req, res) {
```
Comment (groovecoder) [inline controllers/home.js:127]:
```suggestion
function addEmailToBundleWaitlist(req, res) {
```

File: controllers/home.js [hunk 4]
```diff
@@ -146,7 +159,8 @@ module.exports = {
   getAllBreaches,
   getBentoStrings,
   getSecurityTips,
+  getUpgrade,
   protectMyEmail,
-  addEmailToRelayWaitlist,
+  addPrivacyBundleToRelayWaitlist,
```
Comment (groovecoder) [inline controllers/home.js:164]:
```suggestion
  addEmailToBundleWaitlist,
```

File: routes/home.js [hunk 5]
```diff
@@ -18,7 +18,8 @@ router.get("/breaches", getAllBreaches);
 router.get("/security-tips", getSecurityTips);
 router.get("/getBentoStrings", getBentoStrings);
 router.get("/protect-my-email", requireSessionUser, protectMyEmail);
-router.post("/relay-waitlist", jsonParser, requireSessionUser, addEmailToRelayWaitlist);
+router.post("/premium-waitlist", jsonParser, requireSessionUser, addPrivacyBundleToRelayWaitlist);
```
Comment (groovecoder) [inline routes/home.js:21]:
```suggestion
router.post("/premium-waitlist", jsonParser, requireSessionUser, addEmailToBundleWaitlist);
```

File: routes/home.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ const bodyParser = require("body-parser");
 
 const jsonParser = bodyParser.json();
 
-const {home, getAboutPage, getAllBreaches, getBentoStrings, getSecurityTips, protectMyEmail, addEmailToRelayWaitlist, notFound} = require("../controllers/home");
+const {home, getAboutPage, getAllBreaches, getBentoStrings, getSecurityTips, protectMyEmail, addPrivacyBundleToRelayWaitlist, getUpgrade, notFound} = require("../controllers/home");
```
Comment (groovecoder) [inline routes/home.js:9]:
```suggestion
const {home, getAboutPage, getAllBreaches, getBentoStrings, getSecurityTips, protectMyEmail, addEmailToBundleWaitlist, getUpgrade, notFound} = require("../controllers/home");
```

File: public/js/bundle-exp.js [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+          if (targetClasslist.contains("upgrade-go-to-dash")) {
+            window.location.href = serverUrl;
+          }
+        }
+      });
+    });
+
+    document.querySelector(".upgrade-take-survey").addEventListener("click", () => {
+      sendBundleTestPing("Engage", "survey-link");
+      window.open("https://qsurvey.mozilla.com/s3/Firefox-Plus-Unlimited-Promo-Feedback?cohort=xx");
```
Comment (groovecoder) [inline public/js/bundle-exp.js:121]: Is `cohort=xx` correct? Looks like a place-holder value.

**Pull request merged**